test/controllers/announcement_controller_test.exs

```diff
@@ -11,19 +11,33 @@ defmodule ConstableWeb.AnnouncementControllerTest do
     insert(:announcement, title: "Awesome", user: user)
     insert(:announcement, title: "Lame", user: user)
 
-    conn = get conn, Routes.announcement_path(conn, :index, all: true)
+    conn = get(conn, Routes.announcement_path(conn, :index, all: true))
 
     assert html_response(conn, :ok) =~ "Awesome"
     assert html_response(conn, :ok) =~ "Lame"
   end
 
-  test "#index only my announcements are shown by default", %{conn: conn, user: user} do
+  test "#index shows user's announcements when user_id is set", %{conn: conn, user: user} do
+    other_user = insert(:user)
+    insert(:announcement, title: "Awesome", user: user)
+    insert(:announcement, title: "Lame", user: other_user)
+
+    response =
+      conn
+      |> get(Routes.announcement_path(conn, :index, user_id: user.id))
+      |> html_response(:ok)
+
+    assert response =~ "Awesome"
+    refute response =~ "Lame"
```

Comment:
`Lame`

Reply:
I thought so too.

Reply:
https://thoughtbot.slack.com/archives/C024HFHU9/p1558016515265000?thread_ts=1557955114.247600&cid=C024HFHU9

Reply:
I see. I'm guessing that means I should change this?

Reply:
Ha, I thought I was being funny but it looks like it would be good to change this, how would you feel about:

```suggestion
    refute response =~ "Should not appear"
```

Reply:
Updated this (and the test where I had copied the setup from) to avoid using the word "Lame" in fbae53a9b0e7320a0822391e1b996e081835845a. Thanks for the feedback!